NestJS/src/Inbox/inbox.listener.ts

```diff
@@ -27,7 +27,7 @@ export class InboxListener {
 
     async updateInboxList = (message: any):any => {
       this.inboxService.getMessages(message)
-      .then( (messages)  => { this.sendMessage(messages) })
+       .then( (messages)  => { this.sendMessage(messages) })
       .catch( (err) => { this.sendMessage("CAN'T UPDATE INBOX LIST") })
     }
```

Comment:
try to avoid using async/await and Promise simultaneously, just stick to either one